cocos/2d/CCFastTMXLayer.cpp

```diff
@@ -501,11 +501,12 @@ void TMXLayer::updateTotalQuads()
                 
                 float left, right, top, bottom, z;
                 
-                z = getVertexZForPos(Vec2(x, y));
-                auto iter = _indicesVertexZOffsets.find(z);
+                int zPos = getVertexZForPos(Vec2((float)x, (float)y));
+                z = (float)zPos;
```

Comment:
it seems z is not used

Reply:
it used here 
https://github.com/cocos2d/cocos2d-x/blob/a500e7f0ea10c0d485269162022c15ab9cde9a4f/cocos/2d/CCFastTMXLayer.cpp#L527